### PR TITLE
chore: update polkadot/api

### PR DESCRIPTION
**Pull request by tjwelde**
## closes https://github.com/KILTprotocol/ticket/issues/120
This PR updates polkadot/api and adjusts the codebase to meet new api, to support the latest substrate node update.

## How to test:
- The SDK needs to be tested inside the demo client and with the services.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: src/blockchain/Blockchain.ts [hunk 1]
```diff
@@ -112,7 +105,7 @@ export default class Blockchain implements IBlockchainApi {
             log.warn(`Extrinsic error ocurred: ${extrinsicError}`)
             reject(extrinsicError)
           }
-          if (status.type === 'Finalised') {
+          if (status.type === 'Finalized') {
```
Comment (maudnals) [inline src/blockchain/Blockchain.ts]: Suggestion for a follow-up ticket: use consts or Polkadot methods for the statuses over freetext strings

File: src/identity/Identity.spec.ts [hunk 2]
```diff
@@ -11,11 +11,11 @@ describe('Identity', () => {
     expect(alice.seedAsHex).toEqual('0x2f2f416c696365')
 
     expect(alice.address).toEqual(
-      '5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TmTd'
+      '5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu'
```
Comment (maudnals) [inline src/identity/Identity.spec.ts]: why did the address change?

Comment (maudnals) [reply src/identity/Identity.spec.ts]: confused

File: src/ctype/CType.ts [hunk 3]
```diff
@@ -52,10 +52,6 @@ export default class CType implements ICType {
     return CTypeUtils.verifySchema(claim, this.schema)
   }
```
Comment (maudnals) [inline src/ctype/CType.ts]: noice

File: src/identity/Identity.spec.ts (hunk 2 above)
Comment (tjwelde) [reply src/identity/Identity.spec.ts]: The checksum calculation seems to be different now

**Pull request merged**